apps/browser/src/autofill/background/overlay.background.ts

```diff
@@ -512,6 +591,17 @@ export class OverlayBackground implements OverlayBackgroundInterface {
     return this.inlineMenuCiphers.size === 0;
   }
 
+  /**
+   * Stores the credential ids associated with a FIDO2 conditional mediated ui request.
+   *
+   * @param credentials - The FIDO2 credentials to store
+   */
+  private storeInlineMenuFido2Credentials(credentials: Fido2CredentialView[]) {
+    credentials
+      .map((credential) => credential.credentialId)
+      .forEach((credentialId) => this.inlineMenuFido2Credentials.add(credentialId));
```

Comment:
Could we save an iteration here?

```suggestion
      .forEach(({credentialId}) => this.inlineMenuFido2Credentials.add(credentialId));
```

Reply:
Good catch, yeah definitely this needs to be updated to remove the `.map` call. Testing and pushing up some changes for that.